main.py

```diff
@@ -1,10 +1,37 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players:
+        data = json.load(players)
+
+    for nickname, abilities in data.items():
```

Comment:
get_or_create operation can raise exceptions due to validation errors or database issues, so consider adding try/except blocks